lambda-buffers-compiler/test/Test/KindCheck.hs

```diff
@@ -27,38 +25,16 @@ import Test.Tasty (TestTree, testGroup)
 import Test.Tasty.HUnit (assertBool, testCase, (@?=))
 import Test.Tasty.QuickCheck (testProperty)
 import Test.Utils.CompilerInput (
-  compilerInput'doubleDeclaration,
-  compilerInput'doubleDeclarationDiffMod,
   compilerInput'incoherent,
   compilerInput'maybe,
  )
-import Test.Utils.Constructors (_ModuleName)
-import Test.Utils.TyDef (tyDef'maybe)
+import Test.Utils.Constructors (_CompilerInput)
 
 --------------------------------------------------------------------------------
 -- Top Level tests
 
 test :: TestTree
-test = testGroup "Compiler tests" [testCheck, testFolds, testRefl, testMultipleDec]
-
---------------------------------------------------------------------------------
--- Multiple declaration test
-
-testMultipleDec :: TestTree
-testMultipleDec = testGroup "Multiple declaration tests." [doubleDeclaration, passingDoubleDeclaration]
-
-doubleDeclaration :: TestTree
-doubleDeclaration =
-  testCase "Two declarations of Maybe within the same module are caught." $
-    check_ compilerInput'doubleDeclaration
-      @?= Left (P.CompKindCheckError (P.MultipleTyDefError moduleName [tyDef'maybe, tyDef'maybe]))
-  where
-    moduleName = _ModuleName ["Module"]
-
-passingDoubleDeclaration :: TestTree
-passingDoubleDeclaration =
-  testCase "Two declarations of Maybe within different modules are fine." $
-    check_ compilerInput'doubleDeclarationDiffMod @?= Right ()
```

Comment:
Have we added these somewhere else?

Reply:
In progress, 🙂

Reply:
Will be part of a larger 'mutation testing' library.